packages/wrangler/src/dev/miniflare.ts

```diff
@@ -680,6 +678,19 @@ export function buildPersistOptions(
 	}
 }
 
+function buildAssetOptions(config: Omit<ConfigBundle, "rules">) {
+	if (config.experimentalAssets) {
+		return {
+			/** If undefined, this is a static assets only project and there is no user worker */
+			workerName: config.name,
```

Comment:
is there no way for Miniflare to extract the worker name without us passing it as an assets option? Feels like assets options shouldn't necessarily be aware of/have to deal with the user worker name?

Reply:
let's just leave as it is for now